playbooks/roles/ocp-odf-operator/tasks/main.yml

```diff
@@ -0,0 +1,330 @@
+---
+
+ # check if Cluster Health is good
+- name: check all Cluster Operators are available
```

Comment:
We can use [check-cluster-health](https://github.com/ocp-power-automation/ocp4-playbooks-extras/tree/main/playbooks/roles/check-cluster-health) to check the cluster health.

Reply:
Done